test/powershell/engine/Remoting/InvokeCommandRemoteDebug.Tests.ps1

```diff
@@ -0,0 +1,200 @@
+﻿##
+## PowerShell Invoke-Command -RemoteDebug Tests
+##
+
```

Comment:
I don't believe tests are currently run on IoT or Nano Server.  So I feel this shouldn't block the PR merge.